tests/test_lab_dev/test_states.py

```diff
@@ -399,6 +507,36 @@ def test_representation(self, n_modes):
         assert math.allclose(rep.c, [1.0])
 
 
+class TestThermal:
```

Comment:
Here we are not testing the physical properties of the thermal state, perhaps we should? Or shall we do that in integration tests?

Reply:
That is a good question.
Those properties are inherited from `DM`, therefore they are tested in `TestDM` *for some states*.
They may not be tested *for all* the built-in states, but I don't see that as a criticality, because they are agnostic to the state (i.e., they take a representation and do some math, without needing to know the class of the object).

But if you are concerned with things being undertested in this way, I am always up for adding more tests!
And yeah, we should start talking about integration tests, in preparation for Q2